src/Nancy/Response.cs

```diff
@@ -36,14 +36,14 @@ public Response()
             this.Cookies = new List<INancyCookie>(2);
         }
 
-        /// <summary>
-        /// Gets the awaiter.
-        /// </summary>
-        /// <returns><see cref="TaskAwaiter{Response}"/></returns>
-        public TaskAwaiter<Response> GetAwaiter()
-        {
-            return Task.FromResult(this).GetAwaiter();
-        }
+        ///// <summary>

```

Comment:
Oh shit. Commented it out to test and then forgot to delete it. Will fix and update the PR :+1: